<launch>
  
  <arg name="server" default="172.168.0.6"/>

  <node pkg="vrpn_client_ros" type="vrpn_client_node" name="vrpn_client_node" output="screen">
    <rosparam subst_value="true">
      server: $(arg server)
      port: 3883

      update_frequency: 100.0
      frame_id: world

      # Use the VRPN server's time, or the client's ROS time.
      use_server_time: false
      broadcast_tf: true

      # Must either specify refresh frequency > 0.0, or a list of trackers to create
      refresh_tracker_frequency: 1.0
      #trackers:
      #- FirstTracker
      #- SecondTracker
    </rosparam>
  </node>
  <node name="gripper_controller" pkg="handover_controller" type="gripper_controller" output="screen"/>
  <node name="trajectory_generator" pkg="handover_controller" type="trajectory_generator" output="screen"/>
  <node name="robot_arm_controller" pkg="handover_controller" type="robot_arm_controller" output="screen"/>

</launch>